gtsfm/multi_view_optimizer.py

```diff
@@ -24,6 +24,7 @@
 from gtsfm.bundle.bundle_adjustment import BundleAdjustmentOptimizer
 from gtsfm.data_association.data_assoc import DataAssociation
 
+RESULT_METRICS_REACT_PATH = Path(__file__).resolve().parent.parent / "rtf_vis_tool" / "src" / "result_metrics"
```

Comment:
maybe an inline comment here above would help explain why we need to save in 2 places?